Admin/AdminInterface.php

```diff
@@ -656,6 +657,21 @@ public function delete($object);
     /**
      * @param mixed $object
      *
+     * @return int|null
+     */
+    public function getVersion($object);
```

Comment:
@rande If I remove these methods from the AdminInterface, that means that we need to call them directly on the ``ModelManager`` from the ``CRUDController::editAction``. Is it ok ?